android/cli/tests/test-androidmanifest.js

```diff
@@ -40,10 +41,11 @@ describe('AndroidManifest', function () {
 	});
 
 	describe('<application>', function () {
-		var am = new AndroidManifest(path.resolve('./resources/AndroidManifest_application.xml'));
+		const manifest = path.join(__dirname, './resources/AndroidManifest_application.xml');
```

Comment:
<!--
  0 failure: 
  1 warning:  android/cli/tests...
  
  
  DangerID: danger-id-Danger;
  File: android/cli/tests/test-androidmanifest.js;
  Line: 44;
-->

- :warning: android/cli/tests/test-androidmanifest.js line 44 – 'manifest' is assigned a value but never used. (no-unused-vars)